recipes/sqlitecpp/all/conanfile.py

```diff
@@ -24,11 +25,13 @@ class SQLiteCppConan(ConanFile):
         "shared": [True, False],
         "fPIC": [True, False],
         "stack_protection": [True, False],
+        "has_codec": [True, False],
```

Comment:
I think `with_sqlcipher` would be a slightly clearer name for this option.

Reply:
The `has_codec` name is the one used in the cmake of sqlitecpp. I agree that `with_sqlcipher` is easier to understand.